plugins/service-monitor-activemq/src/main/java/com/thinkbiganalytics/service/activemq/ActivemqServiceStatusCheck.java

```diff
@@ -0,0 +1,154 @@
+package com.thinkbiganalytics.service.activemq;
+
+/*-
+ * #%L
+ * service-monitor-activemq
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.thinkbiganalytics.servicemonitor.check.ServiceStatusCheck;
+import com.thinkbiganalytics.servicemonitor.model.DefaultServiceAlert;
+import com.thinkbiganalytics.servicemonitor.model.DefaultServiceComponent;
+import com.thinkbiganalytics.servicemonitor.model.DefaultServiceStatusResponse;
+import com.thinkbiganalytics.servicemonitor.model.ServiceAlert;
+import com.thinkbiganalytics.servicemonitor.model.ServiceComponent;
+import com.thinkbiganalytics.servicemonitor.model.ServiceStatusResponse;
+import com.thinkbiganalytics.service.activemq.util.ActivemqPoolableConnectionProvider;
+
+import org.apache.activemq.jms.pool.PooledConnectionFactory;
+import org.apache.commons.lang3.StringUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+
+import java.util.Arrays;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSException;
+
+/**
+ * Monitor Activemq service and return status to Kylo platform.
+ */
+
+public class ActivemqServiceStatusCheck implements ServiceStatusCheck{
+
+    private static final Logger log = LoggerFactory.getLogger(ActivemqServiceStatusCheck.class);
+
+    private  PooledConnectionFactory activemqPoolConnection = null;
+    private String activemqBrokerUrl = "";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "activemqBrokerUrl" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)